lib/dragonfly_libvips/dimensions.rb

```diff
@@ -13,7 +13,7 @@ def call
         when do_not_resize?
           OpenStruct.new(width: orig_w, height: orig_h, scale: 1)
         when fill_area?
-          OpenStruct.new(width: width, height: height, scale: scale_to_fill)
+          OpenStruct.new(width: fill_width, height: fill_height)
```

Comment:
`scale` does nothing in `thumbnail_image` so we need to change the width/height and add `thumbnail_options[:size] = :both` to allow upscale